homeassistant/components/touchline/climate.py

```diff
@@ -74,17 +74,25 @@ def __init__(self, touchline_thermostat):
         """Initialize the Touchline device."""
         self.unit = touchline_thermostat
         self._name = None
+        self._device_id = None
+        self._controller_id = None
         self._current_temperature = None
         self._target_temperature = None
+        self._min_temp = None
+        self._max_temp = None
         self._current_operation_mode = None
         self._preset_mode = None
 
     def update(self) -> None:
         """Update thermostat attributes."""
         self.unit.update()
         self._name = self.unit.get_name()
+        self._device_id = self.unit.get_device_id()
+        self._controller_id = self.unit.get_controller_id()
         self._current_temperature = self.unit.get_current_temperature()
         self._target_temperature = self.unit.get_target_temperature()
+        self._min_temp = self.unit.get_target_temperature_low()
+        self._max_temp = self.unit.get_target_temperature_high()
```

Comment:
Are the changes in this file relevant to adding a config flow? If not, we should keep this PR focused on the flow.

Reply:
No, they are not relevant for this. I will save it for later :)

Reply:
I also removed this from the PR.